packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+		},
+		"__ignore": {}
+	},
+	"servers": {
+		"__arrayItem": {
+			"url": {
+				"__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$",
+				"__actions": [
+					{
+						"__copy": {
+							"__target": "apiSpecifications.servers[0].url"
+						}
+					}
+				]
+			}
+		}
+	},
+	"paths": {
+		"__keyRegexp": "^\\/[^\\s'\"\\\\]+$",
+		"__objectItem": {
+			"__keyRegexp": "^(get|post)$",
+			"__objectItem": {
+				"__actions": [
+					{
+						"__insert": {
+							"__target": "apiSpecifications.paths.{{1}}.{{2}}.parameters",
+							"__value": []
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[].name",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.method",
+							"__value": "{{2}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.path",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].reservedParameters",
+							"__value": [
+								{
+									"name": "_type"
+								},
+								{
+									"name": "_path"
+								},
+								{
+									"name": "_times"
+								}
+							]
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].fixedOperationParameters",
+							"__value": []
+						}
+					}
+				],
+				"parameters": {
+					"__arrayItem": {
+						"name": {
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[].name"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.name"
+									}
+								}
+							]
+						},
+						"in": {
+							"__regexp": "^(path|query|header|cookie)$",
+							"__actions": [
+								{
+									"__copy": {
+										"__target": "apiSpecifications.paths.{{1}}.{{2}}.{{3}}.in"
+									}
+								},
+								{
+									"__copy": {
+										"__target": "endpoints[_].parameters[_].operationParameter.in"
+									}
+								}
+							]
+						},
+						"schema": {
+							"__conditions": [
+								{
+									"__if": {
+										"default": ".*"
+									},
+									"__then": {
+										"default": {
+											"__actions": [
+												{
+													"__copy": {
+														"__target": "endpoints[_].parameters[_].default"
+													}
+												}
+											]
+										}
+									}
+								}
+							],
+							"__ignore": {}
+						},
+						"__ignore": {}
+					}
+				},
+				"__ignore": {}
+			}
+		}
+	},
+	"components": {
+		"schemas": {
+			"securitySchemes": {
+				"__objectItem": {
+					"type": {
+						"__regexp": "^(apiKey|http)$"
+					},
+					"in": {
+						"__regexp": "^(query|header|cookie)$"
+					},
+					"__conditions": [
+						{
+							"__if": {
+								"type": "^(apiKey|http)$"
+							},
```

Comment:
This should be more strict than an if-condition. `apiKey` and `http` are the only allowed values, so you should first make sure the key exactly matches one of these. See: https://github.com/api3dao/api3-docs/blob/8e41ab2597cb19cb75e4904d6e9b214287c1d738/airnode/ois.md#421-type

Reply:
Not sure if im missing something, but i believe [this](https://github.com/api3dao/airnode/blob/8e97859e8b919e271101f137e24aaba0fa7c8108/packages/validator/templates/OAS2OIS.json#L196) should be already checking it